test/src/main/configurations/MainConfig/ConfigurationMessageStoreSenderAndListener.xml

```diff
@@ -24,14 +24,20 @@
 		</Pipeline>
 	</Adapter>
 	<Adapter name="readIbisstore">
-		<Receiver name="readIbisstoreReceiver">
+		<Receiver name="readIbisstoreReceiver" pollInterval="1">
```

Comment:
When the message is stored to the ibisstore table it gets type 'M'. After a while(*) the message will be picked up by messageStoreListener then the receiver will change the type to 'A'. If I look at scenario01.properties line 25 (in the old file) I see that the select query result comparison is commented out due to this issue. Therefore, I set the pollInterval to 1 so that using one delaypipe is sufficient for messagestorelistener to pick up the message. This way I can have a result file with fixed type in this case 'A' and compare it the result of the select query